doc/guide/Guide_PythonDos-n-Donts.ipynb

```diff
@@ -11,25 +11,6 @@
     "# Coding in Python: Dos and Don’ts"
    ]
   },
-  {
```

Comment:
For the "PEP8 Quickies", syntax highlighting is broken. In the source, I see HTML-style `<pre><code>` commands that should not be there. Can we fix this in this PR?